app/policies/guidance_policy.rb

```diff
@@ -19,6 +19,10 @@ def admin_update?
     user.can_modify_guidance? && guidance.in_group_belonging_to?(user.org_id)
   end
 
+  def index?
+    admin_index?
```

Comment:
nice. we should do this elsewhere too. We have a lot of repeated stuff in the policies